BUILDING.md

```diff
@@ -56,17 +55,15 @@ The tests that should be run in the solution are grouped by project name:
 
 Other test projects contain tests designed to fail purposely for integration tests.
 
-Normally you should be able to run the unit tests directly from within your development IDE of choice (as you would when using NUnit in any other development project). For example, this is what it looks like in JetBrains Rider (2021.1.2) when right clicking on the `AssertEqualsTests` TextFixture:
+You should then be able to run the unit tests directly from within your development IDE of choice against one or all target frameworks (as you would when using NUnit in any other development project). For example, this is what it looks like in JetBrains Rider (2021.1.2) when right clicking on the `AssertEqualsTests` TextFixture:
```

Comment:
I initially, thought that it was odd to write "You should then be able to run the unit tests directly from within...." and then later have "Known Issues and Workarounds", but looking at the resulting page I think this is fine